src/main.ts

```diff
@@ -1,5 +1,6 @@
 import 'source-map-support/register.js';
 import 'dotenv/config';
+import './helpers/parseArgs.js'; // load CLI args
```

Comment:
Is this necessary, considering `parseArgs` is already imported in `events/ready.ts`?